stats/Export.md

```diff
@@ -0,0 +1,34 @@
+# Export API Overview
+The stats library aggregates measurements into Views (which are essentially a collection of
+metrics, each with a different set of labels). All output objects are immutable and read-only.
+The core data types used are:
+* `AggregationData`: contains data from an `Aggregation`.
+* `ViewData`: encapsulates `View` data aggregation.
+
+### AggregationData
+An AggregationData describes the result of data aggregation. The library SHOULD define an
+AggregationData for each Aggregation types defined.
```

Comment:
This does not make sense for all languages--I think C++ is cleaner with no AggregationData (ViewData contains one of several maps, each with a primitive mapped type), and even if it did adopt AggregationData it would be cleaner as a union type with different accessors rather than an interface and an implementation per Aggregation.

As Ramon noted, if we are going to make this a formal API specification, we need to be careful that we are only specifying things at a level of abstraction that translates well across languages--this section on AggregationData, in particular, seems to have Java-esque languages in mind.

Reply:
Not sure I follow your comment here. I think it is wrong in C++ that we combined the View/Aggregation definition with the Data exporting.